src/main/java/core/basesyntax/strategy/impl/SupplyOperationHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.strategy.QuantityIncrementer;
+import java.util.Map;
+
+public class SupplyOperationHandler
+        extends AbstractOperationHandler
+        implements QuantityIncrementer {
+    public SupplyOperationHandler() {
+        super(Operation.SUPPLY);
+    }
+
+    @Override
+    public void processTransaction(Map<String, Integer> fruitQuantityMap,
+                                   FruitTransaction transaction) {
+        incrementByQuantity(fruitQuantityMap, transaction);
+    }
+}
```

Comment:
You have extended `AbstractOperationHandler` but not used any of its functionality in `SupplyOperationHandler`. Please ensure that you are adhering to the SOLID principles. If the abstract class does not provide any shared functionality, consider removing the inheritance or refactoring the code to make use of shared behavior.